client/site-profiler/components/metrics-section/index.tsx

```diff
@@ -16,20 +16,20 @@ const Container = styled.div`
 
 const NameSpan = styled.span`
 	font-family: 'SF Pro Text';
-	color: var( --studio-gray-40 );
+	color: var( --studio-gray-20 );
 	font-size: 16px;
 	margin-bottom: 8px;
 `;
 
 const Title = styled.div`
-	font-family: 'SF Pro Display';
+	font-family: 'SF Pro Text';
 	font-size: 60px;
 	font-weight: 400;
 	line-height: 100%;
 	margin-bottom: 24px;
 
 	span {
-		font-family: 'SF Pro Display';
+		font-family: 'SF Pro Text';
```

Comment:
Similar to my previous comment, I think we can use `$sans` as the default font

Reply:
Updated at https://github.com/Automattic/wp-calypso/pull/90901/commits/83d4f3fe6a6bf14418b516ef3b7fce5d80fcea52